lib/parallel_cucumber/hooks.rb

```diff
@@ -0,0 +1,20 @@
+module ParallelCucumber
+  class Hooks
+
+    @after_batch_hooks ||= []
+
+    class << self
+
+      def after_batch(&proc)
+        raise 'Please provide a valid callback' unless proc.respond_to?(:call)
+        @after_batch_hooks << proc
+      end
+
+      def fire_after_batch_hooks(*args)
+        @after_batch_hooks.each do |hook|
+          hook.call(*args)
```

Comment:
If first hook fails with exception, it will prevent running any further hooks. But at the end worker will ignore hooks error and just log it without failing.

I think this should be clearly documented.